vllm/model_executor/models/mamba.py

```diff
@@ -48,7 +48,7 @@ def __init__(self,
                                 time_step_rank=config.time_step_rank,
                                 use_conv_bias=config.use_conv_bias,
                                 use_bias=config.use_bias,
-                                use_rms_norm=self.is_falcon_mamba,
```

Comment:
I can't find RMS Norm from checkpoints in neither [tiiuae/falcon-mamba-7b](https://huggingface.co/tiiuae/falcon-mamba-7b) nor [tiiuae/falcon-mamba-tiny-dev](https://huggingface.co/tiiuae/falcon-mamba-tiny-dev/tree/main), so I set it to `False`.

Reply:
These should be set to true -- these norms are actually the only difference between falcon-mamba and mamba. 

We should probably handle this in the weight loader -- see discussion here: https://github.com/vllm-project/vllm/pull/9325#discussion_r1798293673

Reply:
Thanks for the explanation! I will revert this and disable the weight loading tracking for mamba temporarily to have a quick fix on failing test.

Let's handle this in a separate PR.